src/payment/__mocks__/defaultState.mockStore.js

```diff
@@ -24,8 +24,18 @@ module.exports = {
     levelOfEducation: null
   },
   payment: {
-    loading: false,
-    loadingError: null,
+    basket: {
```

Comment:
This redux store organization is what I was talking about in slack the other day - more modular/granular.  You'll see it come up elsewhere in this PR.